src/components/form-elements/radio/RadioGroup.jsx

```diff
@@ -0,0 +1,97 @@
+// @flow strict
+
+import * as React from 'react';
+import classNames from 'classnames';
+import {RadioContext} from './useRadioContext';
+import ErrorMessage from '../ErrorMessage';
+import type {RadioColorType} from './Radio';
+
+type DirectionType = 'row' | 'column';
+
+export type RadioGroupPropsType = {
+  children?: React.Node,
+  className?: ?string,
+  color?: RadioColorType,
+  direction?: DirectionType,
+  disabled?: boolean,
+  errorMessage?: string,
+  invalid?: boolean,
+  required?: boolean,
+  name?: string,
+  onChange: (SyntheticInputEvent<HTMLInputElement>) => mixed,
+  value?: string,
+  'aria-labelledby'?: string,
+  'aria-describedby'?: string,
+  ...
+};
+
+const RadioGroup = ({
+  className,
+  children,
+  color,
+  direction = 'column',
+  disabled,
+  errorMessage,
+  invalid,
+  name,
+  required,
+  value,
+  onChange,
+  'aria-labelledby': ariaLabelledBy,
+  'aria-describedby': ariaDescribedBy,
+  ...props
+}: RadioGroupPropsType) => {
+  const [selectedValue, setSelectedValue] = React.useState(value || null);
+  const [lastFocusedValue, setLastFocusedValue] = React.useState(null);
+  const radioGroupClass = classNames('sg-radio-group', className);
+  const radioGroupItemsClass = classNames('sg-radio-group__items', {
+    [`sg-radio-group__items--direction-${String(direction)}`]: direction,
+  });
+  const errorTextId = name ? `${name}-errorText` : undefined;
+
+  const updateValue = (event, value) => {
+    setSelectedValue(value);
+    if (onChange) onChange(event);
+  };
+
+  return (
+    <div
```

Comment:
It would be great for RadioGroup to provide information about the selected value using `aria-activedescendant`:
`aria-activedescendant={id of selected radio item}`